chrome-extension/content.js

```diff
@@ -162,26 +181,31 @@ async function getCategoryDropdown(userId) {
       categoryDropdown.options.add(newOption);
     });
   } catch (error) {
-    // TODO: Handle this error?
+    // TODO: Handle error
```

Comment:
You may want to write a showError(string) function or something down the line

Reply:
Gonna have a filler for when it doesn't load in future pr